client/src/components/Search/Filters/FilterTypes/DateFilter/DateFilter.js

```diff
@@ -0,0 +1,216 @@
+import * as React from 'react';
+import DatePicker from 'react-datepicker';
+import { Form } from 'react-bootstrap';
+import * as moment from 'moment';
+import './DateFilter.scss';
+import Slider from 'rc-slider';
+import {Mode} from './DateEnum';
+
+export default class DateFilter extends React.Component {
+    
+    constructor(props) {
+        super(props)
+        const { data } = props;
+        
+        this.state = {
+            params:{
+                decade: data.params.decade || null,
+                start: data.params.start || null, //TODO - make date dynamic
+                end: data.params.end || null,
+                activeStart: 1870,
+                activeEnd: 1960
+
+            },
+            sliderValues: [1870, 1960],
+            //filterMode: data.config.filterMode || Mode.NONE,
+            dateFilterStart: new Date(),
+            dateFilterEnd: new Date(),
+            filterList: []
+        };
+    }
+
+    async handleFilterChange(name, value) {
+
+        const updatedState = {};
+        const {onChange} = this.props;
+        updatedState[name] = value;
+        await this.setState({
+            params:{
+                ...this.state.params,
+                ...updatedState
+            }
+        });
+        onChange(this.state.params)
+    }
+
+
+    async handleDateFilter(type, value) {
+        switch (type) {
+            case Mode.DECADE:
+                if (value === 'null') value = null;
+                this.handleFilterChange(Mode.DECADE, value);
+                break;
+            case Mode.RANGE:
+                if (value === 'null') value = null;
+                const [ startDate, endDate ] = value;
+                await this.handleFilterChange('start', moment(startDate).format('YYYY-MM-DD'));
```

Comment:
should be yyyy-MM-dd.  Not sure if it'd cause an issue here, but caused one on the backend

Reply:
same on line below this